### ISSUE TITLE
Most of the time it throws an error

### ISSUE THREAD

**Issue by Sachinart** [cckuailong/pocsploit#6]
Which is
```
/usr/local/lib/python3.8/dist-packages/urllib3/connectionpool.py:842: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html
```

![Screenshot_43](https://user-images.githubusercontent.com/18497191/160919653-ed998951-a04f-4e07-b68c-3c5388305293.png)

Please fix it!

It generates lot of unwanted logs.

**Comment by cckuailong** [comment]
It's the requests warnings when ssl verify=False.
I' ll fix it with urllib3.disable_warnings() soon. thank you for report it!

**Comment by cckuailong** [comment]
fixed in [635a77c](https://github.com/cckuailong/pocsploit/commit/635a77ccf80e482f9f8e0731c37ce08de1053098)

**Comment by Sachinart** [comment]
Awesome Thanks!
